pkg/catalog/catalog.go

```diff
@@ -1175,6 +1180,77 @@ func (c *Catalog) GetRange(ctx context.Context, repositoryID, rangeID string) (g
 	return c.Store.GetRange(ctx, graveler.RepositoryID(repositoryID), graveler.RangeID(rangeID))
 }
 
+func (c *Catalog) GetRetentionRules(ctx context.Context, repositoryID string) (*graveler.RetentionRules, error) {
+	return c.Store.GetRetentionRules(ctx, graveler.RepositoryID(repositoryID))
+}
+
+func (c *Catalog) SetRetentionRules(ctx context.Context, repositoryID string, rules *graveler.RetentionRules) error {
+	return c.Store.SetRetentionRules(ctx, graveler.RepositoryID(repositoryID), rules)
+}
+
+func (c *Catalog) PrepareExpiredCommits(ctx context.Context, repository string, previousResultPath string) (string, error) {
```

Comment:
Maybe extract the csv parsing to a designated parser?
It should also handle the schema of this file which is currently static.

Reply:
Done